src/components/radioButton.jsx

```diff
@@ -0,0 +1,30 @@
+import "/src/App.css"
+function RadioButton({ selectedValue, setDeliveryTimeslot, children1, children2 }) {
+
+    const handleChange = (e) => setDeliveryTimeslot(e.target.value)
+    return (
+            <div className="radioButton">
+                <label className="radio-field">
+
+                    <input
```

Comment:
Omdat beide input's geen name-attribuut waarde hebben kun je ze beiden tegelijkertijd aangevinkt hebben

Reply:
Hoi Zwen,

Dankjewel voor je commit. Ik heb het nu zo aangepast. Is dat juist? Als ik het test werkt het prima ;)

  <div className="radioButton">
                <label className="radio-field">

                    <input
                        name="Overdag"
                        type="radio"
                        value="day-time"
                        checked={selectedValue === 'day-time'}
                        onChange={handleChange}
                    />
                    {children1}
                </label>
                <label className="radio-field">
                    <input
                        name="'s Avonds"
                        type="radio"
                        value="evening"
                        checked={selectedValue === 'evening'}
                        onChange={handleChange}
                    />
                    {children2}
                </label>
            </div>